internal/js_library/js_library.bzl

```diff
@@ -214,7 +214,7 @@ def _impl(ctx):
 
     # Don't provide DeclarationInfo if there are no typings to provide.
     # Improves error messaging downstream if DeclarationInfo is required.
-    if len(typings):
+    if len(typings_depsets):
```

Comment:
I don't think this can be right, because it looks like typing_depsets is always at least length one.
(That suggests that even if this PR were green, we are missing a test for "don't provide DeclarationInfo if there are no typings")